postgres_service/postgres_service.py

```diff
@@ -0,0 +1,180 @@
+# this service should be Vaultkeepered
+# uses internal postgres db as source of truth for database hosts and existing databases
+
+import random
+from klein import Klein
+
+from postgres_service.vault import VaultMgr
+from postgres_service.postgres_resource import PostgreSQLDB
+
+
+class PostgresService(object):
+    app = Klein()
+
+    def __init__(self,
+                 vault_token,
+                 service_db,
+                 plans,
+                 vault_addr,
+                 catalog_entry):
+        self.vault_token = vault_token
+        self.plans = plans
+        self.vault_mgr = None
+        self.service_db = service_db
+        self.app = Klein()
+        self.service_db = service_db
+        self.vault_addr = vault_addr
+        self.catalog_entry = catalog_entry
+
+    def setup(self):
```

Comment:
Probably better to raise `NotImplementedError` than `pass`. Could also use some abstract base class but I don't know much about that (better to ask Jeremy).